pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -19,33 +19,243 @@ package dockershim
 import (
 	"fmt"
 
+	dockertypes "github.com/docker/engine-api/types"
+	dockercontainer "github.com/docker/engine-api/types/container"
+	dockerfilters "github.com/docker/engine-api/types/filters"
+
 	runtimeApi "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
 )
 
+const (
+	defaultSandboxImage = "gcr.io/google_containers/pause-amd64:3.0"
+
+	// Various default sandbox resources requests/limits.
```

Comment:
Just curious, will we really use these values as default values in the future? How did we get the default value?